app/views/home/index.html.erb

```diff
@@ -90,10 +90,10 @@
         </span>
       </div>
       <div class="d-md-table-cell v-align-middle col-md-6 direction-md-ltr pr-md-6">
-        <h3 class="alt-h3">Set aside time</h3>
+        <h3 class="alt-h3">Mark your calendar</h3>
 
         <p class="text-gray mb-4">
-        Spend at least two hours on one day every week to work on the open source projects you already use.
+        Set aside two hours per week to work on the open source projects you already use.
```

Comment:
Wrote this somewhere else: I think it's about getting people to take a first tiny step. People who want to spend more than two hours will naturally do so, as they get into it. It's getting people from 0 -> 2 hours that's more important.